pages/resources/application.mdx

```diff
@@ -1732,3 +1732,80 @@ Updates an application's role connection for the user. Returns the updated [appl
 | platform_name?     | string              | The vanity name of the platform a bot has connected (max 50 characters)                                                                                                                                      |
 | platform_username? | string              | The username on the platform a bot has connected (max 100 characters)                                                                                                                                        |
 | metadata?          | map[string, string] | Object mapping [application role connection metadata](#application-role-connection-metadata-object) keys to their `string`-ified value (max 100 characters) for the user on the platform a bot has connected |
+
+<RouteHeader method="GET" url="/applications/{application.id}/managed-links/">
+  Get Application Activity Managed Links
+</RouteHeader>
+
+Returns a list of [activity link](#activity-link-structure) objects for the given application ID.
+
+###### Activity Link Structure
```

Comment:
Move this up and make it a proper object since it's referenced in multiple places. Make sure to update links to reference `-object`